apps/pie-microsite/src/assets/styles/components/_toggle.scss

```diff
@@ -0,0 +1,116 @@
+@use '../fozzie-config' as f;
+
+$toggle-bg: f.$color-container-strong;
+$toggle-radius: f.$radius-rounded-e;
+$toggle-padding: 2px;
+$toggle-item-height: 44px;
+$toggle-item-fw: f.$font-weight-regular;
+$toggle-item-fw--checked: f.$font-weight-bold;
+$toggle-item-padding-y: 10px;
+$toggle-item-padding-x: f.spacing(e);
+$toggle-item-bg: f.$color-container-strong;
+$toggle-item-bg--hover: darken(f.$color-container-strong, f.$color-hover-01);
+$toggle-item-bg--active: darken(f.$color-container-strong, f.$color-active-01);
+$toggle-item-bg--checked: f.$color-interactive-light;
+$toggle-item-bg--checked-hover: darken(f.$color-interactive-light, f.$color-hover-01);
+$toggle-item-bg--checked-focus: f.$color-interactive-light;
+$toggle-item-bg--checked-active: darken(f.$color-interactive-light, f.$color-active-01);
+$toggle-item-color: f.$color-content-interactive-dark;
+$toggle-item-color--hover: f.$color-content-interactive-dark;
+$toggle-item-color--active: f.$color-content-interactive-dark;
+$toggle-item-color--checked: f.$color-content-interactive-dark;
+$toggle-item-elevation: none;
+$toggle-item-elevation--checked: f.$elevation-02;
+
+.c-toggle {
+    display: inline-flex;
+    align-items: center;
+    flex-wrap: nowrap;
+    padding: $toggle-padding;
+    border-radius: $toggle-radius;
+    background-color: $toggle-bg;
+}
+
+.c-toggle-input {
+    &:focus-visible ~ .c-toggle-label {
+        box-shadow: inset 0 0 0 2px f.$color-focus;
+        outline: 2px solid transparent;
+    }
+
+    position: absolute;
+    display: block;
+    overflow: hidden;
+    // IE fallback
+    clip: rect(1px, 1px, 1px, 1px);
+    // modern browsers
+    clip-path: inset(1px);
+    white-space: nowrap;
+}
+
+.c-toggle-item {
+    z-index: 0; // keep inactive items under active ones
+    height: $toggle-item-height;
+
+    &:hover {
```

Comment:
Does :focus need covering?

Reply:
I don't think so no, we have focus-visible applied which I think is all we need

Reply:
OK. I read this as the zindex needs to be above siblings so if it's focus'd was wondering if that would work or not.

Reply:
We've tested the focus states with design and they seem happy, but we will double check